index.html

```diff
@@ -10,58 +10,28 @@
     <script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="
         crossorigin="anonymous"></script>
     <script src="semantic/out/semantic.min.js"></script>
+<<<<<<< HEAD
+
+    <script>
+    $(document).ready(function() {
+        $('#navbar').load('pages/about/navigation.html');
+    });
+</script> 
 </head>
 
-<body>
```

Comment:
This shouldn't have been deleted.

Reply:
@evanhearne it appears that it's been deleted but it's just because of the way that github diff the changes. You can see that the `<body>` tag is still present on line 22 on the right side of the diff. It takes a little bit of time to get used to viewing the diffs.

Reply:
@laurafitzgerald looked at that now, thanks for clearing that up! :)